front/src/applications/operationalStudies/hooks/useProjectImage.ts

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react';
+
+import defaultProjectImage from 'assets/defaultProjectImage/default_project_image.webp';
+import { getDocument } from 'common/api/documentApi';
+
+export const useProjectImage = (imageId?: number | null) => {
+  const [imageUrl, setImageUrl] = useState<string>(defaultProjectImage);
+
+  useEffect(() => {
+    if (!imageId) return;
```

Comment:
We should set the image back to default if the user remove the existing one when editing the project (currently the old image stays displayed until the component is mounted again)
```suggestion
    if (!imageId) {
      setImageUrl(defaultProjectImage);
      return;
    }
```

Reply:
Good catch, thank you.
Done